.github/workflows/snyk_security.yml

```diff
@@ -0,0 +1,60 @@
+# This is a basic workflow to help you get started with Actions
+
+name: snyk_security
+
+# Controls when the workflow will run
+on:
```

Comment:
i think we should still do the snyk test on PRs if they are not from a fork. its a little more work but still most of the repo changes will be from internal PRs and will still be useful to have this in the PR check.

Reply:
included  snyk* jobs conditional checks to internal PRs from this base repo